config/routes.rb

```diff
@@ -80,6 +81,16 @@
       end
     end
 
+    resources :offender_sar_complaints, only: only, controller: 'offender_sar_complaint', as: :case_sar_offender_complaint do
```

Comment:
` controller: 'offender_sar_complaint', as: :case_sar_offender_complaint` why use two separate but similar names? I think it may be better to just stick to either one.

Reply:
I see your point 🤔 

The `as: ... ` routes are set up for namespacing consistency across the correspondence types, by the look of things - but with offender_sar and now with the complaint we've not really followed the same pattern as the rest of them - see `rake routes | grep case_`  

Right now consistency with Offender SAR is the North Star for this work, but we can take a look at it in the next phase maybe?